app/components/events/dropdown-navigation.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const { Component, computed } = Ember;
+export default Component.extend({
+  item: null,
```

Comment:
FYI. Declaration is optional in ember. By default it is null. But it is always recommended to declare so that it's easier to know where it is coming from.